### PR TITLE
Added link to the Constants.Trees.Groups in API documentation and minor readability fixes

### PR DESCRIPTION
**Pull request by poornimanayar**
Hi team,

In the [Trees](https://our.umbraco.com/Documentation/Extending/Section-Trees/trees) page I have 
- added a link to the Constants.Trees.Groups API documentation
- Added an example of tree group in the backoffice
- Improved highlighting of some methods 

Poornima

### CODE REVIEW

File: Extending/Section-Trees/trees.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ public class FavouriteThingsTreeController : TreeController
 The SortOrder controls the order of the custom tree within the Tree Group.
 
 ### Tree Groups
-Tree Groups are a new concept in V8, to enable you to group trees in a section. You provide the alias of the Tree Group name, you wish to add your tree to - see Constants.Trees.Groups for a list of existing group alias. If you add your own alias, you'll need to add a translation key to config/lang/en-US.user.xml to avoid the alias appearing as the header in [square brackets] eg
+Tree Groups are a new concept in V8, to enable you to group trees in a section. You provide the alias of the Tree Group name, you wish to add your tree to - see [Constants.Trees.Groups](https://our.umbraco.com/apidocs/v8/csharp/api/Umbraco.Core.Constants.Trees.Groups.html) for a list of existing group alias. An example of tree groups in the back office would be the *Settings* tree group and the *Templating* tree group in the *Settings* section.
```
Comment (sofietoft) [inline Extending/Section-Trees/trees.md]:
```suggestion
Tree Groups are a new concept in V8, to enable you to group trees in a section. You provide the alias of the Tree Group name, you wish to add your tree to - see [Constants.Trees.Groups](https://our.umbraco.com/apidocs/v8/csharp/api/Umbraco.Core.Constants.Trees.Groups.html) for a list of existing group alias. An example of tree groups in the backoffice would be the *Settings* tree group and the *Templating* tree group in the *Settings* section.
```

File: Extending/Section-Trees/trees.md [hunk 2]
```diff
@@ -130,7 +132,7 @@ The actions on items in an Umbraco Tree will trigger a request to load an Angula
 
 For example clicking on one of the 'Favourite Things' in the custom tree example outlined above will trigger the loading of an 'edit.html' view from the folder: */views/favouriteThingsAlias/edit.html*. The 'Delete' menu item would also load a view from: */views/favouriteThingsAlias/delete.html*
 
-If you're creating a custom tree as part of an Umbraco package/plugin, it's recommended to change the location of the default folder to the app_plugins folder. You achieve this by decorating you mvc *TreeController* with the *PluginController* attribute.
+If you're creating a custom tree as part of an Umbraco package/plugin, it's recommended to change the location of the default folder to the `app_plugins` folder. You achieve this by decorating you mvc *TreeController* with the *PluginController* attribute.
```
Comment (sofietoft) [inline Extending/Section-Trees/trees.md]:
```suggestion
If you're creating a custom tree as part of an Umbraco package/plugin, it's recommended to change the location of the default folder to the `App_Plugins` folder. You achieve this by decorating you mvc *TreeController* with the *PluginController* attribute.
```

**Comment by poornimanayar** [comment]
All done @sofietoft

**Comment by sofietoft** [comment]
Awesome! Thanks 😄

**Pull request merged**